articles/integrations/aws.md

```diff
@@ -271,7 +271,7 @@ The result of calling the delegation endpoint will contain the AWS token in the
 }
 ```
 
-**NOTE:** The Auth0 client libraries simplify calling these endpoints. See our [GitHub repo](https://github.com/auth0/) for the latest SDKs. See an example for client-side JavaScript at: [Delegation Token Request](https://github.com/auth0/auth0.js#delegation-token-request).
+**NOTE:** The [Auth0 client libraries](/libraries) simplify the process of calling these endpoints. See an example for client-side JavaScript at: [Delegation Token Request](https://github.com/auth0/auth0.js#delegation-token-request).
```

Comment:
I would say just replace the link for now. We still haven't removed delegation from our docs and I'm not clear on when this will be done. Just add a link to v7 as Jeff says and mention that it is not supported in the latest version (8).